resources/sass/aante-dark.scss

```diff
@@ -1,23 +1,24 @@
 // $author Ante Aljinovic https://github.com/aljinovic
 // modified dark version
 
-$main-background: invert(#f8f8f8);
-$secondary-background: invert(#f8f8f8);
+@import "aante-light";
 
-$variable-type-color: invert(#06f);
-$variable-type-color-hover: invert(#f00);
+:root {
```

Comment:
I applied this change, although I believe there could be some benefits for using the `:root` space, particularly if we use a short and concise prefix such as `--kt-`.

Would you prefer the following declarations? Otherwise, I'll put the variables in `.kint-rich` as requested.
```scss
:root {
  --kt-variable-type-color: ...
  --kt-variable-type-color-hover: ...
}
```

This would allow the users to easily override Kint themes using custom values later on.

Reply:
Nah just leave it in `.kint-rich` - it's just as hard for a user to write

```css
.kint-rich {
  --variable-type-color: #F00;
}
```

As

```css
:root {
  --kt--variable-type-color: #F00;
}
```

Reply:
Actually what I mean is the ability to override the variable, but sure, I submitted the PR using the `.kint-rich` instead of `:root`

Reply:
You can also do this with `:root .kint-rich` without touching the HTML, but even with that it's not a downgrade.

Before you had to style individual elements since you didn't have access to variables at all. Now you can still do it without more specific selectors if you make sure your css comes after kints (Which is admittedly pretty hard to do but easy enough for actual kint themes)